File: framework/data-jpa/src/test/java/top/bettercode/simpleframework/data/test/resp/UUser.java
package top.bettercode.simpleframework.data.test.resp;

/**
 * @author Peter Wu
 */
public class UUser  {

  private String firstName;
  private String lastName;

  public UUser() {
  }

  public UUser(String firstName, String lastName) {
    this.firstName = firstName;
    this.lastName = lastName;
  }

  public String getFirstName() {
    return firstName;
  }

  public void setFirstName(String firstName) {
    this.firstName = firstName;
  }

  public String getLastName() {
    return lastName;
  }

  public void setLastName(String lastName) {
    this.lastName = lastName;
  }
}

File: data-jpa/src/test/kotlin/top/bettercode/summer/data/jpa/resp/UUser.kt
package top.bettercode.summer.data.jpa.resp

/**
 * @author Peter Wu
 */
class UUser {
    var firstName: String? = null
    var lastName: String? = null

    constructor()
    constructor(firstName: String?, lastName: String?) {
        this.firstName = firstName
        this.lastName = lastName
    }
}
